components/Buttons/src/MDCButton.h

```diff
@@ -89,6 +89,20 @@
  */
 @property(nonatomic, assign) CGSize maximumSize UI_APPEARANCE_SELECTOR;
 
+/**
+ When set, overrides the value returned by |-cornerRadius| and will become the
+ value of the button's layer's cornerRadius. If unset, |-cornerRadius| will return
+ a default value.
+
+ If changing the value of this property while the button is on-screen, it is important to layout
+ the button so that its shadowPath can be updated with the new value.
+
+ @note Do not override the implementation of this property. It will be removed in a future version of
```

Comment:
Based on this comment, I'm not certain if I should be setting this property or setting `layer.cornerRadius` directly. What's the recommendation?

Reply:
The recommendation *will be* to set `layer.cornerRadius` directly. However, doing so today will no have effect since MDCButton assigns the value from `-cornerRadius` in `layoutSubviews`.

Reply:
Ah I see - it may be beneficial to leave that information out of this comment then.

Reply:
Done.